addons/html_editor/static/src/fields/html_field.js

```diff
@@ -77,6 +88,10 @@ export class HtmlField extends Component {
         return this.props.readonly || (this.sandboxedPreview && !this.state.showCodeView);
     }
 
+    get el() {
+        return this?.editor?.editable;
+    }
+
```

Comment:
There is no need to add a getter in the html field (especially if the "dynamic placeholder utility function" is configured inside the plugin).
For example, inside the DynamicPlaceholderPlugin
```js
createDynamicPlaceholder({el: this.editable})
```